drivers/led/lp5569.c

```diff
@@ -83,6 +87,23 @@ static int lp5569_enable(const struct device *dev)
 		return -ENODEV;
 	}
 
+	/* flip the enable pin if specified */
+	if (config->en_pin.port) {
```

Comment:
This is redundant. 
`gpio_is_ready_dt` checks both the port and if initialized.

Reply:
But I don't want to report error if the gpio was not provided.